rurusetto/wiki/templates/wiki/edit_ruleset_wiki.html

```diff
@@ -68,6 +68,17 @@ <h1>Icon & Logo</h1>
                     </div>
                   </div>
             </div>
+            <div class="fieldWrapper">
+              <div class="mb-3 row">
+                 <label class="col-sm-2 col-form-label form-label" for="id_icon">{{ form.dark_icon.label_tag }}</label>
+                 <div class="col-sm-10">
+                     <div class="raleway-font form-current">Current Dark Icon : <a href="{{ form.dark_icon.value.url }}">{{ form.dark_icon.value }}</a></div>
```

Comment:
```suggestion
                     <div class="raleway-font form-current">Current Light Icon : <a href="{{ form.light_icon.value.url }}">{{ form.light_icon.value }}</a></div>
```
Because the website's default theme is dark so it need to change.